### ISSUE TITLE
Serve static content relative to markdown file

### ISSUE THREAD

**Issue by razorheadfx** [razorheadfx/grup#4]
As discussed in #1

TODO:
- Implement serving directory containing the .md file and subdirectories
- Add flag enabling this behaviour
- Add whitelisting for image files
- Update README to include @NickeZ as contributor

**Comment by razorheadfx** [comment]
Hey @NickeZ I'm really sorry about the delay.
Which platform have you tested the #1 on? I compiled and ran it on Windows and I could not see any images in my example files.  Most likely because at this point our server catches all URLs and only serves the stylesheet and the markdown html representation.

I tried to implement URL filtering for files in the directory by hand, but ```simple_server``` was not the most cooperative.
Right now i'm inclined to to remove the```set_static_directory``` line and update the README.
Then, once async is stable implement the filtering logic, should the need arise.
